src/App.jsx

```diff
@@ -38,9 +38,12 @@ const Collections = ({ projects, projectKey, title, currentFilter }) => {
 const App = (props) => {
   const { title, projectKeys, projects, initialFilter } = props;
   const [currentFilter, applyFilter] = useState(initialFilter);
+  const defaultWhite = window.matchMedia(
+    "(prefers-color-scheme: white)"
+  ).matches;
```

Comment:
NICE: This code can be extracted out of `App` to prevent the recalculation on every rerender.